css/index.css

```diff
@@ -1,3 +1,520 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* The use of !important was avoided so Carlos wouldn't cry himself to sleep */
+
+/* 2 Stretch Tasks achieved*/
+
+/* Gotta make some rules here so that everything has a limit */
+.container {
+	width: 950px;
+	margin: auto;
+}
+
+/* Nav bar and header styles */ 
+
+header {
+	height: 60px;
+	width: 950px;
+	position: fixed;
+	top: 0;
+	background-image: linear-gradient(to bottom right, rgb(0, 110, 255), rgb(255, 255, 251));
+	z-index: 10;
+    display: flex;
+
+}
+
+header nav {
+	height: 60px;
+	width: 100%;
+	display: flex;
+	position: relative;
+	line-height: 25px;
+	flex-direction: row;
+	
+}
+
+header nav a {
+	margin: 10px 60px 0 20px;
+	text-decoration: none;
+	justify-content: flex-start;
+	color: rgb(68, 69, 160);
+	line-height: 36px;
+	font-size: 14px;
+}
+
+header .logo {
+	margin-top: 5px;
+	margin-right: 5px;
+	height: 40px;
+    width: auto;
+	justify-content: flex-start;
+}
+
+/* Hands down the most challenging part, at one point the fonts even got wacky on me */
+.main-content {
+	width: 100%;
+	height: 475px;
+	position: relative;
+	display: flex;
+	margin-top: 60px;
+}
+
+.main-content .stack-text {
+	position: absolute;
```

Comment:
Find a way to replace these position absolute properties. There are very limited times when absolute is required, this project should not be one of them.. it has too many downsides in the long run. Please take some time to look them up so that you can understand the issues with it.